script/generate-mime.rb

```diff
@@ -97,7 +98,7 @@ def get_matches(parent)
 puts "  # :nodoc:"
 puts "  EXTENSIONS = {"
 extensions.keys.sort.each do |key|
-  puts "    '#{key}' => '#{extensions[key]}',"
+  puts "    '#{key}' => ['#{extensions[key].join('\',\'')}'],"
 end
```

Comment:
I decided to do it this way to support both formats. Users may have self created tables.rb files which they still want to use.